contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs

```diff
@@ -11,6 +11,9 @@ pub enum ExecuteMsg {
     SetAdmin {
         address: String,
     },
+    TransferAdmin {
+        address: String,
+    },
     SetXCallHost {
```

Comment:
I dont think we need update/ transfer admin. set admin should be enough and should be called by either or owner. @AntonAndell  any thoughts?

Reply:
@gcranju  anton has confirmed that only set admin should be enough and it should be called only by admin . In init message we will have a initial admin address that can set another address as admin.